src/utils.js

```diff
@@ -175,7 +172,76 @@ function taggedString(chaines, ...cles) {
     };
 }
 
+const ALPHANUM = "abcdefghijklmnopqrstuvwxyz0123456789";
+
+function uniqueSlug(size) {
+    let slug = "";
+
+    for (let idx = size; idx--;) {
+        slug += ALPHANUM[(Math.random() * ALPHANUM.lentgth | 0)];
+    }
+
+    return slug;
+}
+
+// eslint-disable-next-line
+const FORBIDDEN_FILENAME_CHAR_REG = /[<>:"\/\\|?*\x00-\x1F]/g;
+
+function filenamify(filename, replacement = "!") {
+    return filename.replace(FORBIDDEN_FILENAME_CHAR_REG, replacement);
+}
+
+// from https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get
+function get(obj, path, defaultValue) {
```

Comment:
Seems slower than _.get. As only 'loadash.get' is required, I don't really think it's a big deal.

Reply:
Yes this method is 4x slower but still very fast
```
lodash get x 4,762,210 ops/sec ±1.67% (87 runs sampled)
jsgeton x 1,240,098 ops/sec ±1.72% (88 runs sampled)
```
IMHO in a non-critical section of code there is no incentive to choose lodash over this method.  

The point is still trying to reduce external dependencies. Even if `lodash.get` is small, it's still an unnecessary dependency